test/javascripts/acceptance/run-query-test.js.es6

```diff
@@ -191,4 +192,19 @@ acceptance("Data Explorer Plugin | Run Query", function (needs) {
       "the chart was rendered"
     );
   });
-});
+
+  function decodeQueryParam(p) {
```

Comment:
Looks like this function isn't being used. Can you remove it?

Reply:
Okay!